Ludown/lib/parseFileContents.js

```diff
@@ -229,19 +225,40 @@ module.exports.parseFile = function(fileContent, log)
                                     LUISJsonStruct.utterances.push(utteranceObject);
                                 }
                                 
-                                LUISJsonStruct.patterns.push(patternObject);
                                 
                                 if(utterance.includes("{")) {
                                     // handle entities
                                     var entityRegex = new RegExp(/\{(.*?)\}/g);
                                     var entitiesFound = utterance.match(entityRegex);
                                     entitiesFound.forEach(function(entity) {
                                         entity = entity.replace("{", "").replace("}", "");
+                                        patternAnyEntitiesInUtterance = true;
                                         addItemIfNotPresent(LUISJsonStruct, LUISObjNameEnum.PATTERNANYENTITY, entity);
                                     });
                                 }
                             }
                         });
+                        if(entitiesInUtterance.length !== 0) {
+                            // examine entities and push
+                            var utteranceObject = {
+                                "text": updatedUtterance,
+                                "intent":intentName,
```

Comment:
just happened to notice a missing whitespace while skimming email :)

`"intent": intenName"`